custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -337,6 +344,92 @@ def handle_exception(self, exception, repeat_record):
                                 "followup_nikshay_registered", "followup_nikshay_error")
 
 
+@RegisterGenerator(NikshayRegisterPrivatePatientRepeater, 'case_xml', 'XML', is_default=True)
```

Comment:
Right! i used xml because this eventually sends out an xml request.
It looks like a better way out instead of adding another variable unless it conflicts with the implementation in Base payload generator.